src/chrome/content/rules/Onlyoffice.org.xml

```diff
@@ -13,6 +13,6 @@
 	<target host="dev.onlyoffice.org" />
 	<target host="cloud.onlyoffice.org" />
 
-	<rule from="^http://(www\.)?onlyoffice\.org/" to="https://$1onlyoffice.com/" />
+	<rule from="^http://(www\.)?onlyoffice\.org/" to="https://www.onlyoffice.com/" />
```

Comment:
```
	<rule from="^http://onlyoffice\.org/" to="https://www.onlyoffice.com/" />
```

Reply:
https://travis-ci.org/EFForg/https-everywhere/jobs/593026985#L335

Reply:
oops

Reply:
Let's just try to whitelist this